### PR TITLE
fix: handle STX assets title

### PR DESCRIPTION
**Pull request by csgui**
Pull Request to fix an issue with the PR #739 .

It wasn't being considered the case where the project has more than one asset to be displayed.

**Before the fix on this PR:**
image

**After the fix on this PR:**
image

image

If any other place needs attention, please let me know. Thanks!

### CODE REVIEW

**Review by hugocaillard — CHANGES_REQUESTED** [review]
My bad for missing this as well. Thanks for spotting it!

Could you make it so "uStx" is still the first token to be displayed (to be in the first column) as it used to be?
Thanks!

**Comment by csgui** [comment]
Hey @hugocaillard ,

> Could you make it so "uStx" is still the first token to be displayed (to be in the first column) as it used to be?

Yes, I can reorder the columns but it wasn't like that before. STX column was always the last one as far as I remember.

Should I work on the column reorder anyway?

Thanks!

File: components/clarity-repl/src/repl/session.rs [hunk 1]
```diff
@@ -963,7 +963,15 @@ impl Session {
         let accounts = self.interpreter.get_accounts();
         if accounts.len() > 0 {
             let tokens = self.interpreter.get_tokens();
-            let headers = vec!["Address".to_string(), "uSTX".to_string()];
+            let mut headers = vec!["Address".to_string()];
+            for token in tokens.iter() {
+                if String::from("STX").eq(&String::from(token)) {
```
Comment (hugocaillard) [inline components/clarity-repl/src/repl/session.rs:968]: You should be able to do `if token == "STX"` here, or is their any reason why not?

**Comment by hugocaillard** [comment]
Oh right no need to re-order then. Just left a comment but otherwise it seems fine like that 👍

File: components/clarity-repl/src/repl/session.rs (hunk 1 above)
Comment (csgui) [reply components/clarity-repl/src/repl/session.rs]: The operator `==` relies on the `eq` function to do the comparison. Thus, this add one more underlying function call to the stack. But I think there is no need to worry with that... it's only a "muscular" memory.

And I should agree that the infix notation (1 == 1) is more readable. :-)... changed.

Comment (hugocaillard) [reply components/clarity-repl/src/repl/session.rs]: Oh okay, didn't know about that, thanks for the explanation

**Review by hugocaillard — APPROVED** [review]
LGTM!
Thanks @csgui

**Pull request merged**